### PR TITLE
add install wrapper and trigger for GH

### PR DESCRIPTION
**Pull request by denisok**
https://jira.percona.com/browse/PMM-9985

Sequence of landing:

1. https://github.com/Percona-Lab/pmm-submodules/pull/2583
2. https://github.com/percona/pmm-qa/pull/488
3. https://github.com/Percona-Lab/jenkins-pipelines/pull/1601

### CODE REVIEW

File: pmm/kubernetes-cluster-staging.groovy [hunk 1]
```diff
@@ -146,10 +146,7 @@ pipeline {
                                 set -o errexit
                                 set -o xtrace
                                 sudo yum -y install curl
-                                sudo curl -Lo /usr/local/sbin/minikube https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube-linux-amd64
-                                sudo chmod +x /usr/local/sbin/minikube
-                                sudo ln -s /usr/local/sbin/minikube /usr/sbin/minikube
-                                alias kubectl='minikube kubectl --'
+                                /srv/pmm-qa/install_k8s_tools.sh --minikube --kubectl --sudo
```
Comment (puneet0191) [inline pmm/kubernetes-cluster-staging.groovy:149]: `/srv/pmm-qa/pmm-tests/install_k8s_tools.sh` 
should be the path

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: pmm/kubernetes-cluster-staging.groovy (hunk 1 above)
Comment (denisok) [reply pmm/kubernetes-cluster-staging.groovy:149]: Thanks!

**Review by puneet0191 — APPROVED** [review]

**Review by beatahandzelova — APPROVED** [review]

File: pmm/kubernetes-cluster-staging.groovy [hunk 2]
```diff
@@ -146,10 +146,7 @@ pipeline {
                                 set -o errexit
                                 set -o xtrace
                                 sudo yum -y install curl
-                                sudo curl -Lo /usr/local/sbin/minikube https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube-linux-amd64
-                                sudo chmod +x /usr/local/sbin/minikube
-                                sudo ln -s /usr/local/sbin/minikube /usr/sbin/minikube
-                                alias kubectl='minikube kubectl --'
+                                /srv/pmm-qa/pmm-tests/install_k8s_tools.sh --minikube --kubectl --sudo
```
Comment (nikita-b) [inline pmm/kubernetes-cluster-staging.groovy:149]: We have Jenkins agent where kubectl already was pre-installed:
https://github.com/percona/pmm-infra/blob/main/packer/ansible/agent.yml
Can we migrate to this agent?

I don't really like the approach where we download tools every time :(

**Review by nikita-b — APPROVED** [review]

File: pmm/kubernetes-cluster-staging.groovy (hunk 2 above)
Comment (denisok) [reply pmm/kubernetes-cluster-staging.groovy:149]: maybe, from other point - how often we update the image and the tool in the image is also downloaded (not from distr) so you would need to bump version quite often.

**Pull request merged**